src/app_engine/analytics.py

```diff
@@ -75,7 +75,8 @@ def report_event(self, event_type, room_id=None, time_ms=None,
       event[LogField.FLOW_ID] = flow_id
 
     if client_type is not None:
-      event[LogField.CLIENT_TYPE] = client_type
+      client_type_name = ClientType.Name.get(client_type, str(client_type))
```

Comment:
Can we check types of all the input here or in analytics_page? The client may send any type and we should validate.